packages/flutter_tools/lib/src/dart/pub.dart

```diff
@@ -16,6 +16,43 @@ import '../cache.dart';
 import '../globals.dart';
 import 'sdk.dart';
 
+/// Represents Flutter-specific data that is added to the `PUB_ENVIRONMENT`
+/// environment variable and allows understanding the type of requests made to
+/// the package site on Flutter's behalf.
+// DO NOT update without contacting kevmoo.
+// We have server-side tooling that assumes the values are consistent.
```

Comment:
Is the server side tooling public code somewhere? It would be nice to actually point to the code rather than to a single person here, in case e.g. you're on vacation or something.

Reply:
It's not public. We could put a `go` link here. Reasonable?

Reply:
Sure. Maybe something like:

```
// Google runs server-side tooling that depends on these exact values.
// Please avoid updating this code without having someone at Google
// update the corresponding back-end code at the same time.
// Contact @kevmoo for details. Googlers, see go/foobar.
```